### DISCUSSION TITLE
importError: cannot import name handlers

### DISCUSSION THREAD

**Discussion by bitteryuki** [Monika-After-Story/MonikaModDev#10326] [QA - tech support + general]
<img width="645" alt="Screenshot 2024-10-30 at 17 08 29" src="https://github.com/user-attachments/assets/0685dce3-40bb-4005-ae7d-615b26d2ce32">

i have a mac, and tried to download the mod, but ive only got errors. only once i was able to play the mod on my mac, but when i tried to start a new game, everything crashed.

**Comment by Booplicate** [comment]
Avoid using steam
